numpydoc/__init__.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import division, absolute_import, print_function
 
-__version__ = '0.6.0'
+__version__ = '0.7.dev'
```

Comment:
Good idea. This is not a valid PEPxxx (forgot the spec number) version number though - can you please make it ``0.7.0.dev0`` ?